WcaOnRails/app/assets/javascripts/competitions.js

```diff
@@ -3,6 +3,26 @@ onPage('competitions#edit, competitions#update, competitions#admin_edit, competi
     $('.wca-registration-options').toggle(this.checked);
   }).trigger('change');
 
+  $('input[name="competition[guests_entry_fee_enabled]"]').on('change', function() {
+    $('.wca-guests-entry-fee-enabled').toggle(this.checked);
```

Comment:
This don't look right to me. It looks like you're reacting to a change in `guests_entry_fee_enabled`, and when that happens, you're toggling `guests_entry_fee_enabled`. Are you sure this is right?

Reply:
Maybe I didn't completely get how this works. I'll make sure to review and fix this!